server/models/Document.ts

```diff
@@ -733,6 +734,41 @@ class Document extends ArchivableModel<
     return null;
   }
 
+  /**
+   * Find many documents by their id.
+   * Supports fetching the membership data when "userId" is specified in the options
+   *
+   * @param ids array of document ids
+   * @param options FindOptions
+   * @returns A promise resolving to the list of documents
+   */
+  static async findByIds(
+    ids: string[],
+    options?: Omit<FindOptions<Document>, "where"> &
+      Omit<AdditionalFindOptions, "rejectOnEmpty">
+  ): Promise<Document[]> {
+    const userId = options?.userId;
+    const rest = omit(options, "userId");
```

Comment:
```suggestion
    const { userId, ...rest } = options;
```

preferred? You would have to provide a default value of empty object for `options` is all

Reply:
`options` is an optional param here. Do we want to make it required?
I was thinking since it's a common `findByIds` method, this could potentially be used as `findByIds(list_of_ids)` in the future.

Reply:
`options = {}` solves it